open-metadata-implementation/access-services/subject-area/subject-area-client/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/client/nodes/DefaultSubjectAreaNodeClients.java

```diff
@@ -49,14 +43,14 @@ public SubjectAreaNode(SubjectAreaRestClient subjectAreaRestClient, String... pa
                     ctor.setAccessible(true);
                     final AbstractSubjectAreaNode newInstance =
                             (AbstractSubjectAreaNode) ctor.newInstance(subjectAreaRestClient);
-                    cache.put(newInstance.type(), newInstance);
+                    cache.put(newInstance.resultType(), newInstance);
                 }
             } catch (NoSuchMethodException
                     | IllegalAccessException
                     | InstantiationException
                     | InvocationTargetException e) {
                 throw new ExceptionInInitializerError(
-                        "During initialization SubjectAreaNode an error has occurred - "
+                        "During initialization `DefaultSubjectAreaNodeClients` an error has occurred - "
```

Comment:
please use the standard way of doing messages for subject area -add a new entry into SubjectAreaErrorCode and refer to that